src/controllers/queries/index.ts

```diff
@@ -359,16 +318,21 @@ export class QueriesController extends HTMLController {
       await Promise.all(
         new Array(size).fill({}).map((_, i) => {
           if (partial.connectionIds && partial.productIds && partial.quantities) {
-            req.log.debug('validated partial query and submitting DRPC request to %s', partial.connectionIds[i])
-            return this.submitQueryRequest({
+            req.log.debug('submitting DRPC request to %s connection', partial.connectionIds[i])
+
+            return this.submitDrpcRequest({
               parentId: queryRow.id,
+              method: 'submit_query_request',
               connectionId: partial.connectionIds[i],
               log: req.log,
               localQuery: {
+                query: 'Scope 3 Carbon Consumption',
                 quantity: parseInt(partial.quantities[i]),
                 productId: partial.productIds[i],
-                emissions,
+                emissions, // TODO investigate this before asking for review, should not be needed?
```

Comment:
correct this should not be needed

Reply:
oops, thought I';ve removed glad left a comment, sadly, this is another leftover

Reply:
it has been removed along with property and unit tests

Reply:
to be fair might going to push along with drpc unit tests shortly